src/drv_oculus_rift/rift-hmd-radio.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Oculus Rift CV1 Radio
+ * Copyright 2016 Philipp Zabel
+ * Copyright 2019 Jan Schmidt
+ * SPDX-License-Identifier:	BSL-1.0
+ */
+#ifndef __RIFT_HMD_RADIO_H__
```

Comment:
I think we don't add underscores around these.

Reply:
Fixed.